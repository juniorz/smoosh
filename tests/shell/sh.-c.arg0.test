cat > scr <<EOF
echo "i am \$0, hear me roar"
EOF
$TEST_SHELL -c '. "$0"' ./scr