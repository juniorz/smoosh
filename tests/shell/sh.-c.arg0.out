i am ./scr, hear me roar
